### ISSUE TITLE
Scenario Logger gives misleading versions, timestamps of Raiden in the node logs

### ISSUE THREAD

**Issue by agatsoh** [raiden-network/scenario-player#364]
Description
=========
In some of the scenarios especially the mfee scenarios the `run-000.log` file of the node logs from the SP  report that they are using raiden from 26th September 2019 or 5th October 2019 in the scenario player runs which is misleading.

```
{"raiden": "0.100.5a1.dev504+g68448ac97", "raiden_db_version": 23, "python_implementation": "CPython", "python_version": "3.7.3", "system": "Linux 64bit 4.15.0-64-generic", "architecture": "x86_64", "distribution": "source", "event": "Starting Raiden", "logger": "raiden.ui.runners", "level": "info", "timestamp": "2019-10-05 04:38:12.780377"}
```

```
{"raiden": "0.100.5a1.dev445+gf40adc88e", "raiden_db_version": 23, "python_implementation": "CPython", "python_version": "3.7.3", "system": "Linux 64bit 4.15.0-62-generic", "architecture": "x86_64", "distribution": "source", "event": "Starting Raiden", "logger": "raiden.ui.runners", "level": "info", "timestamp": "2019-09-26 06:53:06.746398"}
```
This happens almost regularly

Edit(17.10.2019)
Closer analysis of the log runs 16th Oct 2019 mfee4 scenario.
[Scenario_player_Logs-mfee4_combined_fees-1-latest-2019-10-16.tar.gz](https://github.com/raiden-network/scenario-player/files/3738952/Scenario_player_Logs-mfee4_combined_fees-1-latest-2019-10-16.tar.gz)

1. `run-000.log` of `node_0_003`
* The raiden version **"raiden": "0.100.5a1.dev445+gf40adc88e"**
* The timestamps occuring in the entire file  are **"timestamp": "2019-09-26**

2. `run-000.stdout` of `node_0_003`
* The raiden version `Welcome to Raiden, version ` **`0.100.5a1.dev445+gf40adc88e!`** same as that in `run-000.log`

3. `scenario-player-run_mfee4_combined_fees_2019-10-16T05:39:05.log`
* It gives the correct date of `2019-10-16`.
`{"version": "local", "event": "Using Raiden version", "logger": "scenario_player.node_support", "level": "info",` **`"timestamp": "2019-10-16 05:39:06.802424"}`**
* `timestamp` throughout are log file are correct for that particular date
`"timestamp": "2019-10-16`

Most probably logger picks up wrong date and version for raiden.

Expected Behavior
===============
Scenario player logger should each time notify correct raiden version and correct timestamp in all the log files that it creates.

**Comment by nlsdfnbch** [comment]
This may be an issue with the pack-logs command. Could you double check the logs on the SP instances manually? (serverd at port 8000). That would at least explain the timestamp issue.

As for the raiden version, the SP can only log what raiden tells it :man_shrugging:

**Comment by christianbrb** [comment]
Fixed by #365
